rootfs/etc/cont-init.d/30-vpncfg

```diff
@@ -26,6 +26,7 @@ echo -e "/vpn/auth.conf" >/var/run/s6/container_environment/AUTH_CFG
 echo -e "/var/run/openvpn.pid" >/var/run/s6/container_environment/OVPN_PID
 
 [[ ! -f "/vpn/config/updated" ]] &&
+  mkdir /vpn/config &&
```

Comment:
Can you add -p so that it won't complain if the directory exists. I bind a volume to /vpn/config in my setup and didn't catch this. Thanks!